README.md

```diff
@@ -80,6 +80,62 @@ Under `.addPackage(new MainReactPackage())`:
 .addPackage(new ReactVideoPackage())
 ```
 
+#### Windows
+
+Make the following additions to the given files manually:
+
+**windows/myapp.sln**
+
+Add the `ReactNativeVideo` project to your solution.
+
+1. Open the solution in Visual Studio 2015
+2. Right-click Solution icon in Solution Explorer > Add > Existing Project...
+3. Select `node_modules\react-native-video\windows\ReactNativeVideo\ReactNativeVideo.csproj`
+
+**windows/myapp/myapp.csproj**
+
+Add a reference to `ReactNativeVideo` to your main application project.
+
+Using Visual Studio 2015:
+
+1. Right-click main application project > Add > Reference...
+2. Check `ReactNativeVideo` from Solution Projects.
```

Comment:
Best to give the user only visual studio version and not the XML.

Perhaps more of  Rightclick  project > Add to project > Check box